spacy/pipeline/entityruler.py

```diff
@@ -348,6 +348,45 @@ def clear(self) -> None:
             self.nlp.vocab, attr=self.phrase_matcher_attr, validate=self._validate
         )
 
+    def remove(self, ent_id: str) -> None:
+        """Remove a pattern by its ent_id if a pattern with this ent_id was added before
+
+        ent_id (str): id of the pattern to be removed
+        RETURNS: None
+        DOCS: https://spacy.io/api/entityruler#remove
+        """
+        label_id_pairs = [
+            (label, eid) for (label, eid) in self._ent_ids.values() if eid == ent_id
+        ]
+        if not label_id_pairs:
+            raise ValueError(Errors.E1023.format(ent_id=ent_id))
+        created_labels = [
+            self._create_label(label, eid) for (label, eid) in label_id_pairs
+        ]
+        self.phrase_patterns = defaultdict(
+            list,
+            {
+                label: val
+                for (label, val) in self.phrase_patterns.items()
+                if label not in created_labels
+            },
+        )  # remove the patterns from self.phrase_patterns
```

Comment:
Maybe this is a Black formatting thing, but wouldn't this be easier to read if these comments were above the start of the expression rather than the end?

Reply:
Ah yes, I originally put the comments to the same line then Black moved them to next line. I'll carry them above expressions.